lib/_http_server.js

```diff
@@ -562,7 +562,7 @@ function resOnFinish(req, res, socket, state, server) {
 
   res.detachSocket(socket);
   req.emit('close');
-  res.emit('close');
+  process.nextTick(emitCloseNT.bind(res));
```

Comment:
You could also do `process.nextTick(emitCloseNT, res)` to avoid the bind and accept `res` as an arg in your emitCloseNT function

Reply:
this will be better.